### DISCUSSION TITLE
Ativação do Discussions, Adição de Membro

### DISCUSSION THREAD

**Discussion by pedroleone** [pedroleone/osr-srd#10] [General]
Salve pessoal!
Estou ativando a parte de discussões, acho que pode ser produtivo para discutir coisas que não são de fato Issues.
Além disso, incluí o @nerun como membro do projeto. Ele já está trabalhando incansavelmente na revisão e tradução. Muito obrigado @nerun!

**Comment by nerun** [comment]
Salve pessoal! Orgulhoso de me juntar ao projeto!
